src/makeInfinityAdder.js

```diff
@@ -3,8 +3,24 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(a = 0) {
+  let sum = a;
+
+  function add(b) {
+    if (b === undefined) {
```

Comment:
The check `if (b === undefined)` is correct for determining when to return the accumulated sum. Ensure that this behavior aligns with the intended use of the function.